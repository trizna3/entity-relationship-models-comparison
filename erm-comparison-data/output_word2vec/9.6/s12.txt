account-customers;accounts
life insurance-life insurance
real estate insurance-real estate insurance
contract-contracts
persons;addresses-persons
real estate types-address
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estates
GENERALIZATION_TO_11_ASSOCIATION:S-insured-persons
CONTRACT_11_ASSOCIATION:E-persons-addresses
CONTRACT_11_ASSOCIATION:S-customers-accounts
