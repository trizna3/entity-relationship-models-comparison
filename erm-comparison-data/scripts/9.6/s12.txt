entity set customers {
};

entity set persons {
	"name"
	"surname"
	"brith number"
};

entity set insured {
};

entity set accounts {
	"username"
	"password"
};

entity set contracts {
	"date from"
	"date to"
	"fee"
};

entity set "life insurance" {
	"fatal amount"
	"consequence amount "
	"work incapacity dose"
};

entity set "real estate insurance" {
	"crash insurance"
	"elemental disaster insurance"
};

entity set "real estates" {
	"address"
	"type"
	"living area size"
	"basement size"
};

entity set "insured data" {
	"name"
	"surname"
	"birth number"
};

association between 
{
	customers ONE 
	accounts ONE
}{};

association between 
{
	persons ONE 
	contracts MANY
}{};

association between 
{
	"life insurance" ONE 
	"insured data" ONE
}{};

association between 
{
	"real estate insurance" ONE 
	"real estates" ONE
}{};

generalization from customers to persons;

generalization from insured to persons;

generalization from "real estate insurance" to contracts;

generalization from "life insurance" to contracts;
