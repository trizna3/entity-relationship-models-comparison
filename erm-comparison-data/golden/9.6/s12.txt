account-accounts
contract-contracts
life insurance-life insurance
real estate insurance-real estate insurance
persons-persons
addresses-address
real estate types-type
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-type-real estates
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estates
GENERALIZATION_TO_11_ASSOCIATION:S-customers-persons
CONTRACT_11_ASSOCIATION:S-customers-persons